### DISCUSSION TITLE
Pre-processing data

### DISCUSSION THREAD

**Discussion by Suparat-Scheu** [Rappsilber-Laboratory/XiSearch#67] [General]
Hi,
I have questions following the paper (Reliable identification..), Lenz included denoising  before convert to mgf file. Could you explain on how to denoise the peak before convertion to mgf.file? Any guidlines to denoise the peak? I think I might try to do this to reduce search space, apart from reduce the low abundance proteins on the database.
Currently, I performed BS3-crosslinking in bacterial lysate, fowllowed by SEC enrichment and LC/MSMS. I would like to be sure I do it correctly when I convert the raw files to mgf files. For my case, what type of filter parameters would you recommend before convert to mgf file?     From what I did, I selected only peak picking with parameter vendor on ms level=2.  
When will be a case to use peak picking at ms level =1? 

Thank you very much

Suparat

**Comment by lutzfischer** [comment]
Denoising can be done during mgf convertion  (e.g. MSConvert hase a denoise option). But depending on the resolution of the MS2 denoising can help - but also hurt. So for very high resolution MS2 I would not denoise. For lower resolutution I would sue something like 10 to 15 peaks per 100Th window,

xiSEARCH at the moment does not consider any MS1 information besides the precursor m/z denoted for the MS2 and also when you convert top mgf the MS1 spectra are not included - so ms level=1 makes no sense/difference there.

**Reply by cxdummies** [reply]
Hi Lutz,

I would be interested to know more about the relationship between the resolution and the decision to use a denoising filter or not. Could you elaborate more on this? What are the advantages and disavantages of using denoising? How do you define high- and low-resolution MS2 in this context?

Thank you.

**Reply by lutzfischer** [reply]
What I mean here with high resolution is something up to 5ppm MS2 error. When you have a like this or higher each peak - even low intense ones - that can be matched is more likely going to help more then it introduces noise. Meaning it does add more confidence to a correct match then it will likely help a random match. While if you have lower resolutions and have to search with something like 20 ppm the balance starts shifting towards supporting random matches more often then the actual correct match. 

There is no real clear cut - that I am aware of. But if you search with something like 15 or 20ppm MS2 tolerance denoising the spectra to something like 12 to 15 peaks per 100da helps. On the other hand when searching with 5ppm MS2 tolerance no denoising seems to yield better results.

But these are experience values based on Orbitrap instruments - so if you have something else (that has a different noise profile) you might have to adjust that a bit.

**Reply by cxdummies** [reply]
Thank you for the detailed explanation.

If I want to apply the preprocessing step in MaxQuant as described in your Xi-MPA paper, how can I deactivate the denoising step? It is described in the paper that one can set 20 top peaks per 100 Da, but how can one skip it?

Another question, if I may. In that paper, the "deisotoping" option is inactivated. Does xiSEARCH take care of the deisotoping step by itself? What would happen to the result if one deisotoped the MS2 spectra during preprocessing?

**Reply by lutzfischer** [reply]
You can "skip" the denoising step by simply defining a large value. E.g something like 100 or 1000. So it would do a denoising step, only it would not actually denoise anything.

As for deisotoping - the reason that it should be disabled, is that xiSEARCH makes use of the charge states as part of the internal candidate processing. For one thing it takes the actual fragment mass is used and also the information whether or not a isotop cluster is detected for a peak to begin with.  So I would expect that your results would suffer if you deisotope/decharge the spectra before xiSEARCH.

**Comment by grandrea** [comment]
Hello @cxdummies , our current preprocessing pipeline can be found at https://github.com/Rappsilber-Laboratory/preprocessing . Currently denoising is disabled by default. I hope this helps!
